### ISSUE TITLE
Inconsistent page template slugs/names

### ISSUE THREAD

**Issue by samsour** [fictionco/fiction#144]
**Reason for this issue**
While creating a custom page template I noticed that a custom template was not applied because of the template loader looking for a wrong template-slug ().
Instead of the template slug set in `index.js` the loader searches for a template slug that is the template name in _kebab-case_. "Custom Landingpage" ->_custom-landingpage_ instead of the wanted slug _landingpage_.

Also see [this forum post](https://factor.dev/topic/5e56f4695a9997002c376502/how-to-set-custom-page-template-settings).

**Steps to Reproduce**
Using @factor/core 1.4.2.

Add a custom page template in the index.js
```js
import { addPageTemplate } from "@factor/templates"

addPageTemplate({
  name: "Name",
  slug: "slug",
  component: () => import("./templates/Landingpage")
})
```

Add `templates/Landingpage.vue`:
```js
import { stored } from "@factor/api";
import Vue from "vue";
export default Vue.extend({
  computed: {
    post() {
      return stored("post") || {}
    },
    settings() {
      return this.post.settings || {}
    }
  },
  templateSettings() {
    return [
      {
        input: "text",
        label: "Heading",
        description: "Primary page heading",
        _id: "pageHeading",
        default: "Landing Page Template"
      }
    ]
  }
})
```

Now select the page template for a new page in the factor dashboard. If you now view the page, the default template will be loaded instead of the landing page template.

![image](https://user-images.githubusercontent.com/37221148/75452869-4b708d80-5973-11ea-8fe2-c83a61db53ad.png)

**Comment by arpowers** [comment]
Still can't seem to figure it out, in your admin, are the page template values the same as the slug? 

I don't know why `templateLocation` is getting set to `name`

<img width="296" alt="Screen Shot 2020-02-27 at 1 25 59 PM" src="https://user-images.githubusercontent.com/308134/75483799-c67e7780-5964-11ea-982f-bc5795c17cfc.png">

**Comment by samsour** [comment]
Ah I think I mixed things up with the screenshots... Originally I wanted to show that the template name is also the value of the option field.

***This better describes my problem:***

```js
addPageTemplate({
  name: "Landingpage Name",
  slug: "landingpage-slug",
  component: () => import("./templates/Landingpage")
})
```

![image](https://user-images.githubusercontent.com/37221148/75492173-7ed50b80-59b7-11ea-809f-cc4578d28a05.png)

Instead of `landingpage-slug` the `templateLocation` will look for `landingpage-name`.

**Comment by arpowers** [comment]
Found the issue, this should be fixed in next patch release 1.4.3
